apps/web/public/locales/en/app.json

```diff
@@ -58,7 +58,7 @@
   "preferences": "Preferences",
   "previousMonth": "Previous month",
   "register": "Register",
-  "requiredString": "“{name}” is required",
+  "requiredString": "{name} is required",
```

Comment:
I don't think this change is necessary or relevant to this PR. Can you please revert this?